app/models/grda_warehouse/tasks/chronically_homeless.rb

```diff
@@ -61,48 +103,8 @@ def run!
       @chronically_homeless = []
       @client_details = {}
       extra_work = 0
-      @clients.each_with_index do |client_id, index|
-        debug_log "Calculating chronicity for #{client_id}"
-        # All chronically homeless must also be disabled
-        next unless disabled?(client_id)
-        # remove any cached calculations from the previous client
-        reset_for_batch()
-        adjusted_homeless_dates_served = residential_history_for_client(client_id: client_id)
-        homeless_months = adjusted_months_served(dates: adjusted_homeless_dates_served)
-        homeless_months_size = if homeless_months.size > 36 then 36 else homeless_months.size end
-        # debug_log homeless_months.inspect
-        debug_log "Found #{homeless_months.size} homeless months"
-        if homeless_months.size >= 12
-          disabled = disabled?(client_id)
-          debug_log "Client disabled? #{disabled.inspect}"
-          if disabled
```

Comment:
This `if disabled` block seemed completely redundant to me, since we checked in on like 67 already. Removed from the refactored method.

Reply:
I agree.  Thanks.